tron/commands/cmd_utils.py

```diff
@@ -194,3 +198,13 @@ def setup_logging(options):
         format='%(name)s %(levelname)s %(message)s',
         stream=sys.stdout,
     )
+
+
+def suggest_possibilities(word, possibilities, max_suggestions=3):
```

Comment:
Could we increase the value of max_suggestions a little bit, like 10? I feel we have many similar names at Tronfig files.